src/app/login/login.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router'; // Importar o Router
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  email = '';
```

Comment:
Utilize tipagem forte para melhorar a segurança do tipo e a legibilidade do código. Por exemplo:
```ts
public emailFormControl: FormControl<string>;
```

OBS: mesmo que implicitamente seja public, quando n colocamos qual o tipo do modificador de acesso, é interessante para a legibilidade do seu código EXPLICITAR não só o tipo de valor que o valor tem, mas tbm seu modificador:

`public, private, protected`

Caso não esteja acostumado com essa nomenclatura, é um conceito de POO muito importante para o desenvolvimento, já que angular é orientado a objetos.
Aqui esta UM SUPER resumo bem generalista de quando utilizar cada um:

# Modificadores de Acesso em TypeScript/Angular

## public
- Acessível de qualquer lugar, dentro ou fora da classe.
- É o modificador padrão se nenhum for especificado.
- Use para propriedades e métodos que devem ser livremente acessíveis.

## private
- Acessível apenas dentro da própria classe onde foi definido.
- Não pode ser acessado de fora da classe ou por classes filhas.
- Use para encapsular detalhes internos de implementação.

## protected
- Acessível dentro da classe e em classes que herdam dela.
- Não pode ser acessado de fora da classe ou suas subclasses.
- Use para permitir acesso em uma hierarquia de classes, mas não globalmente.